javascript-if/main.js

```diff
@@ -0,0 +1,61 @@
+/* exported isUnderFive,
+            isEven,
+            startsWithJ,
+            isOldEnoughToDrink,
+            isOldEnoughToDrive,
+            isOldEnoughToDrinkAndDrive,
+            categorizeAcidity,
+            introduceWarnerBro
+ */
+function isUnderFive(number) {
+  if (number < 5) {
+    return true;
+  } else return false;
```

Comment:
When creating an `else` statement, you should add another set of brackets. So in this example, your code should look like the following 

<img width="275" alt="Screen Shot 2023-02-01 at 11 36 58 AM" src="https://user-images.githubusercontent.com/71413368/216145541-03abf97a-8fce-4d00-a300-43de965a7163.png">